integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/binary/TestBinaryDataType.scala

```diff
@@ -158,6 +158,63 @@ class TestBinaryDataType extends QueryTest with BeforeAndAfterAll {
         assert(true)
     }
 
+
+    test("Unsupport DICTIONARY_INCLUDE for binary") {
+
+        sql("DROP TABLE IF EXISTS binaryTable")
+        val exception = intercept[MalformedCarbonCommandException] {
+            sql(
+                """
+                  | CREATE TABLE binaryTable(
+                  |     id int,
+                  |     name string,
+                  |     city string,
+                  |     age int,
+                  |     binaryField binary)
+                  | STORED BY 'carbondata'
+                  | tblproperties('dictionary_enable'='true','dictionary_include'='binaryField')
```

Comment:
dictionary_enable is not a supported table property, please remove.

Reply:
done